### PR TITLE
particle demo

### PR DESCRIPTION
**Pull request by wwwtyro**
@ibgreen @mikolalysenko 

Adds a particle demo for #24 

gif: http://i.imgur.com/kMZWjNd.gifv

Screenshot: 

particles-example

### CODE REVIEW

File: examples/particles/index.html [hunk 1]
```diff
@@ -0,0 +1,303 @@
+<html>
+
+  <head>
+    <title>Particle demo</title>
+    <meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link href="../examples.css" type="text/css" rel="stylesheet" media="screen" />
```
Comment (Qix-) [inline examples/particles/index.html]: You self-close 7 but not 8 here.

File: examples/particles/index.html [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+    <script src="http://d1n0x3qji82z53.cloudfront.net/src-min-noconflict/ace.js" type="text/javascript" charset="utf-8"></script>
+    <script type="text/javascript" src="../../dist/LumaGL.js"></script>
+    <script type="text/javascript" src="../examples.js"></script>
+    <script type="text/javascript" src="index.js"></script>
+
+
+
+    <script id="scene-vs" type="x-shader/x-vertex">
+
```
Comment (Qix-) [inline examples/particles/index.html]: Why so many extra lines?

**Comment by Qix-** [comment]
For the record this looks really cool.

**Comment by wwwtyro** [comment]
@Qix- Thanks for the review and kudos. :+1: :grinning:

**Pull request merged**